importer/importer/builder.py

```diff
@@ -389,6 +391,42 @@ def save_image(image_source_url):
         return 0
 
 
+def get_product_accountability_period(product_id: str) -> int:
+    product_endpoint = "/".join([fhir_base_url, "Group", product_id])
+    response = handle_request("GET", "", product_endpoint)
+    if response[1] != 200:
+        logging.error(
+            "Error while attempting to get the accountability period from product : "
+            + product_id
+        )
+        logging.error(response[0])
+        return -1
+
+    json_product = json.loads(response[0])
+    product_characteristics = json_product["characteristic"]
+    for character in product_characteristics:
+        if (
+            character["code"]["coding"][0]["display"]
+            == "Accountability period (in months)"
+        ):
```

Comment:
probably best to use the `code.coding.system` and `code.coding.code`, since those are less likely to be moving targets.

```
export const smartRegisterCodeSystem = 'http://smartregister.org/codes';
export const accountabilityCharacteristicCode = '67869606';

export const accountabilityCharacteristicCoding = {
  system: smartRegisterCodeSystem,
  code: accountabilityCharacteristicCode,
  display: 'Accountability period (in months)',
};

```